static/src/js/actions/project.js

```diff
@@ -295,6 +296,16 @@ export const getProjectCollections = () => async (dispatch, getState) => {
                   form
                 }
               }
+              variables {
```

Comment:
You are going to get test failures from this on my PR I have to do something similar